functions/Remove-DbaDbData.ps1

```diff
@@ -0,0 +1,166 @@
+
+    .PARAMETER SqlCredential
+        Login to the target instance using alternate Windows or SQL Login Authentication. Accepts credential objects (Get-Credential).
+
+    .PARAMETER Database
+        The database(s) to process. This list is auto-populated from the server. If unspecified, all user databases will be processed.
+
+    .PARAMETER ExcludeDatabase
+        The database(s) to exclude. This list is auto-populated from the server.
+
+    .PARAMETER InputObject
+        Enables piped input from Get-DbaDatabase
+
+    .PARAMETER WhatIf
+        Shows what would happen if the command were to run. No actions are actually performed.
+
+    .PARAMETER Confirm
+        Prompts you for confirmation before executing any changing operations within the command.
+
+    .PARAMETER EnableException
+        By default, when something goes wrong we try to catch it, interpret it and give you a friendly warning message.
+        This avoids overwhelming you with "sea of red" exceptions, but is inconvenient because it basically disables advanced scripting.
+        Using this switch turns this "nice by default" feature off and enables you to catch exceptions with your own try/catch.
+
+    .NOTES
+        Tags: Role, Database
+        Author: Jess Pomfret (@jpomfret), jesspomfret.com
+
+        Website: https://dbatools.io
+        Copyright: (c) 2018 by dbatools, licensed under MIT
+        License: MIT https://opensource.org/licenses/MIT
+
+    .LINK
+        https://dbatools.io/Remove-DbaDbData
+
+    .EXAMPLE
+        PS C:\> Remove-DbaDbData -SqlInstance localhost -Database dbname
+
+        Removes all data from the dbname database on the local default SQL Server instance
+
+    .EXAMPLE
+        PS C:\> Remove-DbaDbData -SqlInstance mssql1 -ExcludeDatabase DBA -Confirm:$False
+
+        Removes all data from all databases on mssql1 except the DBA Database. Doesn't prompt for confirmation.
+
+    .EXAMPLE
+        PS C:\> $svr = Connect-DbaInstance -SqlInstance mssql1
+        PS C:\> $svr | Remove-DbaDbData -Database AdventureWorks2017
+
+        Removes all data from AdventureWorks2017 on the mssql1 SQL Server Instance, uses piped input from Connect-DbaInstance.
+
+    .EXAMPLE
+        PS C:\> Get-DbaDatabase -SqlInstance mssql1 -Database AdventureWorks2017 | Remove-DbaDbData
+
+        Removes all data from AdventureWorks2017 on the mssql1 SQL Server Instance, uses piped input from Get-DbaDatabase.
+
+    #>
+    [CmdletBinding(SupportsShouldProcess, ConfirmImpact = 'High')]
+    param (
+        [DbaInstance[]]$SqlInstance,
+        [PSCredential]$SqlCredential,
+        [string[]]$Database,
+        [string[]]$ExcludeDatabase,
+        [parameter(ValueFromPipeline)]
+        [object[]]$InputObject,
+        [string]$Path = (Get-DbatoolsConfigValue -FullName 'Path.DbatoolsExport'),
+        [switch]$EnableException
+    )
+
+    begin {
+        $null = Test-ExportDirectory -Path $Path
```

Comment:
Need to add this at the start of the process block so it fails properly since this will try to stop the execution of the command.

```if (Test-FunctionInterrupt) { return }```

Reply:
added